src/rpcrawtransaction.cpp

```diff
@@ -343,6 +344,8 @@ UniValue createrawtransaction(const UniValue& params, bool fHelp)
             "      ...\n"
             "    }\n"
             "3. locktime                (numeric, optional, default=0) Raw locktime. Non-0 value also locktime-activates inputs\n"
+            "4. opt into RBF            (boolean, optional, default=false) Allow this transaction to be replaced by a transaction with heigher fees\n"
+
```

Comment:
IMHO this really isn't necessary,  the RBF rules are very clear such that if `sequence` is specified at all, then it will be used.
Its as much to just add the `sequence` field per input and just omit `opt-in RBF` parameter entirely.

IMHO.